cmd/utils/flags.go

```diff
@@ -1957,6 +1957,22 @@ func RegisterEthService(stack *node.Node, cfg *ethconfig.Config) (ethapi.Backend
 	if err != nil {
 		Fatalf("Failed to register the Ethereum service: %v", err)
 	}
+
+	// register the live tracer
+	if cfg.VMTrace != "" {
+		var traceConfig json.RawMessage
+		if cfg.VMTraceJsonConfig != "" {
+			traceConfig = json.RawMessage(cfg.VMTraceJsonConfig)
+		}
+		tracer, apis, err := tracers.LiveDirectory.New(cfg.VMTrace, traceConfig, backend.APIBackend)
+		if err != nil {
+			Fatalf("failed to create tracer %s: %v", cfg.VMTrace, err)
+		}
+		backend.BlockChain().SetLogger(tracer)
+		if apis != nil {
+			stack.RegisterAPIs(apis)
```

Comment:
We do have access to the `node.Node` object in eth backend constructor. Why did you move this here?

Reply:
I want to pass the Backend to the live tracer, the same as `tracers.APIs(backend.APIBackend)`